autoPyTorch/pipeline/components/setup/network_backbone/ResNetBackbone.py

```diff
@@ -327,11 +338,14 @@ def forward(self, x: torch.FloatTensor) -> torch.FloatTensor:
             if self.config["multi_branch_choice"] == 'shake-shake':
                 x1 = self.layers(x)
                 x2 = self.shake_shake_layers(x)
-                alpha, beta = shake_get_alpha_beta(self.training, x.is_cuda)
+                alpha, beta = shake_get_alpha_beta(is_training=self.training,
+                                                   is_cuda=x.is_cuda,
+                                                   method=self.config['shake_alpha_beta_method'])
```

Comment:
`function`, but not `method`?
If it is not hard to change, `shake_shake_update_func` is more precise based on the original paper.

Reply:
sure